pontoon/base/templates/locale.html

```diff
@@ -13,7 +15,13 @@
 			<p>{{ locale.team_description|safe }}</p>
 		</div>
 	{% endif %}
+	{{ Menu.submenu((
+		(url('pontoon.locale', locale.code), 'Projects', ''),
+		(url('pontoon.locale.contributors', locale.code), 'Contributors', 'contributors')
+	), request.path.split('/')[2]|default('')) }}
```

Comment:
Menu (div class="submenu") shall be moved above the form tag, to keep it consistent with contributors page. This will also fix some spacing issues.